Documentation/doc/resources/1.8.13/menu_version.js

```diff
@@ -23,6 +23,15 @@
   ];
 
   function build_select(current_version) {
+    if( current_version == 'master') {
+      let top_elt = document.getElementById("top");
+
+      let first_element = top_elt.childNodes[0];
+      let new_div = document.createElement("p");
+      new_div.innerHTML = '⚠️ This documentation corresponds to the <a style="font-familly: monospace;" href="https://github.com/CGAL/cgal/tree/master">master</a> branch of CGAL, that is not yet released.';
```

Comment:
```suggestion
      new_div.innerHTML = '⚠️ This documentation corresponds to the <a style="font-familly: monospace;" href="https://github.com/CGAL/cgal/tree/master">master</a> development branch of CGAL. It might diverge from the official releases.';
```

Reply:
Thanks. I have used that suggestion.